File: dart/24.05.2024.dart
import 'dart:io';

void main(){
  print ('ingrese su nomnre');
  String? N = stdin.readLineSync();

  print ('ingrese apellido');

  String? AP = stdin.readLineSync();

  String completename = N! + AP!; 

  print ('ingrese un numero:');
  int? n1 = int.parse(stdin.readLineSync()!);

  print('ingrese un numero:');
  int? n2 = int.parse(stdin.readLineSync()!);

  print ('ingrese un numero:');
  int? n3 = int.parse(stdin.readLineSync()!);

  int Sum = n1 + n2 + n3;
  double prom = Sum / 3;

  print ('el resultado del promedio es: $prom');
  print ('tu nombre completo es: $completename');
}

File: ruby/24.05.2024.rb
puts "ingrese su nombre"
n = gets.chomp 
puts "ingrese apellido"
ap = gets.chomp 

puts "ingrese un numero:"
n1 = gets.chomp.to_i
puts "ingrese un numero:"
n2 = gets.chomp.to_i
puts "ingrese un numero:"
n3 = gets.chomp.to_i
sum = n1 + n2 + n3 
prom = sum / 3
puts "tu nombre completo es #{n} #{ap}"
puts "el promedio es #{prom}"